src/Hl7.Fhir.Core/Serialization/BaseFhirParser.cs

```diff
@@ -32,9 +33,10 @@ public BaseFhirParser(ParserSettings settings = null)
 #pragma warning restore CS0618 // Type or member is obsolete
             };
 
-        public Base Parse(ITypedElement element) => element.ToPoco(buildPocoBuilderSettings(Settings));
+        // TODO BIG_COMMON
+        //public Base Parse(ITypedElement element) => element.ToPoco(buildPocoBuilderSettings(Settings));
 
-        public T Parse<T>(ITypedElement element) where T : Base => element.ToPoco<T>(buildPocoBuilderSettings(Settings));
+        public T Parse<T>(ITypedElement element, ModelInspector inspector) where T : Base => element.ToPoco<T>(inspector, buildPocoBuilderSettings(Settings));
```

Comment:
Shouldn't the inspector be a private member on BaseFhirParser, initialized by the constructor?